### PR TITLE
Teams

### PR DESCRIPTION
**Pull request by kate-walters**
Teams and members - let me know if anything needs cleaning up.

### CODE REVIEW

**Comment by rluck0419** [comment]
"Prefer single-quoted strings when you don't need string interpolation or special symbols." - even in the schema? shut up hound-bot

it does make a few good suggestions, one in particular stands out - take a look at the duplicate 'redirect_to :back'

the extra line after "private" is good practice too - but the rest we could probably more or less ignore. should be :+1: after that

**Comment by MattRice12** [comment]
👍

File: app/assets/stylesheets/teams.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the teams controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (jah2488) [inline app/assets/stylesheets/teams.scss]: Empty file. boo

File: app/assets/javascripts/teams.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (jah2488) [inline app/assets/javascripts/teams.coffee]: Empty file. boo

**Comment by jah2488** [comment]
My 👍 follows what @rluck0419 said.

**Pull request merged**